tests/core/engine_adapter/test_integration.py

```diff
@@ -2035,8 +2035,6 @@ def test_dialects(ctx: TestContext):
             None,
             {
                 "default": pd.Timestamp("2020-01-01 00:00:00+00:00"),
-                # https://github.com/pymssql/pymssql/issues/649
```

Comment:
doesn’t this depend on the version? if you’re gonna remove this you may have to pin the version, if not you’ll need backwards compatibility right?

Reply:
To my knowledge, no sqlmesh behavior will change - just the test. Our plan was to have people handle the bytes -> python conversion themselves if it ever came up.

Reply:
ah ok great